modules/deploy/pages/deployment-option/cloud/managed-connectors/create-gcs-connector.adoc

```diff
@@ -104,6 +104,9 @@ a| The template for file names on GCS. Supports `{{ variable }}` placeholders fo
 - `timestamp:unit=yyyy\|MM\|dd\|HH` (the timestamp of the record)
 - `key` (when used, other placeholders are not substituted)
 
+There are two sets of these placeholders supported, `topic,partition,start_offset,timestamp` and `key`, and they are exclusive.
+For the former set, the `topic,partition,start_offset` are required placeholders.
```

Comment:
```suggestion
There are two sets of placeholders supported, `topic,partition,start_offset,timestamp` and `key`, which  are exclusive.
For the former set, `topic,partition,start_offset` is a required placeholder.
```

Reply:
@tomasz-sadura Is the placholder name in the last sentence correct? It soes not match the previous sentence. Shouldn't it read `topic,partition,start_offset,timestamp` instead?

Reply:
Taken your suggestion and rephrased a little bit, please check out if it's more clear now.